src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp

```diff
@@ -1304,6 +1304,36 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_com_microsoft_fused_conv_relu_z_input) {
     test_case.run_with_tolerance_as_fp();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_range_positive_step) {
+    const auto function = onnx_import::import_onnx_model(file_util::path_join(ov::test::utils::getExecutableDirectory(),
```

Comment:
This needs to be changed as well: [example](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/onnx_import_com_microsoft.in.cpp#L1159-L1171).
Please update and merge with master to align with current changes